app/main.py

```diff
@@ -1 +1,67 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self,
+                 name: str) -> None:
+        super().__init__(name)
+
+        for i in ["JavaScript", "HTML", "CSS"]:
+            self.learn_skill(i)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+
+    def __init__(self,
+                 name: str) -> None:
+        super().__init__(name)
+
+        for i in ["Python", "SQL", "Django"]:
+            self.learn_skill(i)
```

Comment:
JFYI: you can use list.extend() here instead of `learn_skill`.

Reply:
What practice is better?

Reply:
I think in this case it's better to use extend, because you are hardcoding the values anyway. It will just look better.